src/backend/transcriber/src/main/java/com/amazonaws/transcriber/LifecycleInfoPersister.java

```diff
@@ -0,0 +1,67 @@
+package com.amazonaws.transcriber;
+
+import com.amazonaws.utils;
+import kong.unirest.Unirest;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
+import software.amazon.awssdk.services.dynamodb.model.PutItemRequest;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class LifecycleInfoPersister {
+
+    private static final Logger logger = LogManager.getLogger(LifecycleInfoPersister.class);
+
+    private static final String METADATA_ENDPOINT = utils.requiredEnvVar("ECS_CONTAINER_METADATA_URI");
+    private static final String TASK_METADATA_ENDPOINT = METADATA_ENDPOINT + "/task";
+
+    private String taskArn;
+    private DynamoDbClient client = DynamoDbManager.getClient();
+    private TranscriberConfig config = TranscriberConfig.getInstance();
+    private String ddbTableName = config.tasksDynamoDbTable();
+
+    private String mediaUrl;
+
+    LifecycleInfoPersister(String mediaUrl) {
+        this.taskArn = Unirest
+            .get(TASK_METADATA_ENDPOINT)
+            .asJson()
+            .getBody()
+            .getObject()
+            .getString("TaskARN");
```

Comment:
Should we also update the cfn template for changing the references from `TaskId` to `TaskARN`?

Reply:
That was what my change was meant to do but it got lost when I did my rebase thing.